_docs-sources/foundations/ci-cd/pipelines.md

```diff
@@ -0,0 +1,64 @@
+# Pipelines
+
+Gruntwork Pipelines is a framework for deploying Infrastructure as Code changes and managing requests for and creation of AWS accounts, with first class support for [Terragrunt](https://terragrunt.gruntwork.io/). Pipelines enables you to use your preferred CI tool* to detect changes to Infrastructure as Code Infrastructure Units (IU)(individual modules containing `terragrunt.hcl` files) and securely run `terragrunt plan` and `terragrunt apply` jobs on each Infrastructure Unit.
+
+Pipelines runs as a step in your CI tool, determining the types of changes that were made (e.g., adding, changing, or deleting a module) and the action to take based on if your change was made in a Pull Request (PR) or in a commit to your main branch (e.g., the PR has been merged). As an example, creating a pull request with changes that add a new IU containing an AWS RDS instance in your `dev` account would detect the new module and run a `terragrunt plan` in the directory containing the IU.
+
+*_GitHub Actions is the only currently supported CI tool_
+
+## Components of Pipelines
+
+Pipelines is a single binary that runs as a step in your CI tool. To perform the actions that Pipelines detects, each account will need an AWS IAM Role that allows GitHub Actions to assume it using OIDC. This role is automatically provisioned in new accounts that are provisioned when [adding an account](../accounts/add-account.md) using the Account Factory.
```

Comment:
I wasn't sure either, initially I thought it should be uppercase because it's a first class concept for us (i think?) but it's not really a _product_ just a concept. I could go either way.